src/config.rs

```diff
@@ -75,6 +75,7 @@ fn memory_mb_for_cf(is_raft_db: bool, cf: &str) -> usize {
 
 #[derive(Clone, Serialize, Deserialize, PartialEq, Debug)]
 #[serde(default)]
+#[serde(deny_unknown_fields)]
```

Comment:
Seems we can no longer downgrade if there is a new config exist?

Reply:
You'd need to comment out that config manually. The error message will tell you the name of the offending field.

Reply:
seem TiDB has the same problem now. is it ok?
@shenli

Reply:
yes, I think so.